packages/react-themeless-provider/src/utils/createCSSStyleSheetFromTheme.ts

```diff
@@ -0,0 +1,13 @@
+import { createCSSRuleFromTheme } from '@fluentui/react-components';
+import type { PartialTheme } from '@fluentui/react-theme';
```

Comment:
```suggestion
import type { PartialTheme } from '@fluentui/react-components';
```

this is exported from `@fluentui/react-components`